packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
@grebaldi regarding commands/actions, I don't really thing it makes sense to call action creators and actions differently, in our case they are basically exactly the same thing, they are hardly mapped/coupled.
It's clear that sometimes we use actions to tell the app to do something, and sometimes we just use them as signals to reflect that something has already happened and to listen to that signal via sagas.
E.g. `finishDocumentInitiazation` doesn't really work because document initialization has already been finished by this point, and we use an action merely to notify others of the already happened fact. If not for the sagas we would not even need that action...

One idea could be to use some different mechanism for those event-like actions, that would make them stand our from the stream of simple actions, or maybe bundle them all into a special sub-reducer e.g., e.g. maybe move them all into `System/`?